### PR TITLE
ADCM-2824 add string to check UI artefacts, make error_msg more clear

### PR DESCRIPTION
**Pull request by SofiaTal**

### CODE REVIEW

File: python/api/group_config/serializers.py [hunk 1]
```diff
@@ -264,14 +264,18 @@ def check_value_unselected_field(cc, nc, gk, spec, obj):
...
                         ):
                             continue
 
                     if not v and k in cc and k in nc and cc[k] != nc[k]:
-                        raise AdcmEx('GROUP_CONFIG_CHANGE_UNSELECTED_FIELD')
+                        msg = (
+                            f"Value of {k} field is different in current and new config."
+                            f" Current: ({cc[k]}),new: ({nc[k]})"
+                        )
+                        raise AdcmEx('GROUP_CONFIG_CHANGE_UNSELECTED_FIELD', msg)
```
Comment (a-alferov) [inline python/api/group_config/serializers.py:278]: Add pls `log.info()`

**Review by a-alferov — CHANGES_REQUESTED** [review]

**Review by kuhella — COMMENTED** [review]
Did you check this in attached data? 
Try to change any parameter (with active checkbox) and save it& is it ok?

**Review by DanBalalan — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Comment by SofiaTal** [comment]
make build

**Comment by SofiaTal** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @SofiaTal! Build has been launched!

**Pull request merged**